### ISSUE TITLE
How to get started with App Manager?

### ISSUE THREAD

**Issue by MuntashirAkon** [AMCommunity/TipsAndTricks#6]
App Manager offers a lot of features, most of which are not known to an ordinary Android user. Therefore, it is quite normal for an ordinary user to find App Manager quite irrelevant if not utter useless. For example, some people find App Manager unusable in no-root mode although a lot of time has been spent on making it usable in no-root mode. [App Manager user manual](http://muntashirakon.github.io/AppManager/) is a great way to get started with Android ecosystem. It shall definitely arise new questions, and if you are willing to invest enough time and has patience, it will also help you develop a more investigative mind.

Those who are already familiar with some features should be able get started quite fast. App Manager’s user manual is fully available offline within the application itself. Therefore, a willing user can look up a feature any time at any place without leaving the application. As App Manager offers a large amount of features, the possibilities of utilising them are also infinite. This is why many people ask for specific features only to find that they are already available in the application. You should remember that App Manager has been made more generalised than most other applications that you usually use. Therefore, features that are quite highlighted in other applications may appear to be only a minor feature in App Manager.

**Comment by AbhiTheModder** [comment]
@MuntashirAkon I think we should pin this issue at top

**Comment by shuvashish76** [comment]
Some projects support multiple version documentations e.g [Syncthing](https://docs.syncthing.net/), [Kodi](https://kodi.wiki/index.php?title=Main_Page&action=history)... We should clearly mention that AM documentation is available for "latest Stable release" only, though we already mentioned date/version in manual.

Change suggestion:
- [App Manager user manual](http://muntashirakon.github.io/AppManager/) [_available for "latest [Stable](https://github.com/amcommunity/TipsAndTricks/issues/17) release" only_] is a great way to get started with Android ecosystem.
- Or add it as a separate "NOTE".

---

![Screenshot_20231016-181830](https://github.com/amcommunity/TipsAndTricks/assets/48226312/fbc0f459-6803-483d-aad1-8f5ee22b667a)

Just like Syncthing, in the manual add ~User Manual~ "**Written for** _v3.1.2 Stable_", to make it clear it's written for App version not User Manual version or something...

**Comment by AbhiTheModder** [comment]
> Some projects support multiple version documentations e.g [Syncthing](https://docs.syncthing.net/), [Kodi](https://kodi.wiki/index.php?title=Main_Page&action=history)... We should clearly mention that AM documentation is available for "latest Stable release" only, though we already mentioned date/version in manual.
> 
> Change suggestion:
> - [App Manager user manual](http://muntashirakon.github.io/AppManager/) [_available for "latest [Stable](https://github.com/amcommunity/TipsAndTricks/issues/17) release" only_] is a great way to get started with Android ecosystem.
> - Or add it as a separate "NOTE".
> 
> ---
> 
> ![Screenshot_20231016-181830](https://github.com/amcommunity/TipsAndTricks/assets/48226312/fbc0f459-6803-483d-aad1-8f5ee22b667a)
> 
> Just like Syncthing, in the manual add ~User Manual~ "**Written for** _v3.1.2 Stable_", to make it clear it's written for App version not User Manual version or something...

@shuvashish76 Great Idea, Surely there are needs for changes in Manual considering `contact` section etc aren't updated yet even though it's been a while since **Matrix** and **Mastodon** (well it's new) has already been introduced but I _don't think there's any need_ for Doing documentation for each type of releases as you've suggested

From you comment it's clear that you've read the [Releases Diff.](https://github.com/amcommunity/TipsAndTricks/issues/17) Section and through that you can be clear that there aren't such major changes to these releases so it's not required yet.

**Comment by shuvashish76** [comment]
> Surely there are needs for changes in Manual considering contact section etc aren't updated yet even though it's been a while since Matrix and Mastodon (well it's new) has already been introduced.

Yes but no relevance to this specific issue, needs separate issue preferably on AM project as it has "Docs improvement" issue template for it.

> I _don't think there's any need_ for Doing documentation for each type of releases as you've suggested.

🤷🏿‍♂️ I didn't suggest for all version/type documentations. So many things change so frequently before stable release, so makes no sense to add them all to docs; it's not possible anyway.

> that there aren't such major changes to these releases so it's not required yet.

If you've tried alpha releases you can easily see lot of things has been changed.

Reasons:
1. I suggested here because this particular Tip&Tricks (TT) will be listed in top when AM release it's official AM_Community website.
2. When users will try versions other than **latest stable** they'll find many feature documentations missing and unnecessarily report on GH. e.g: I've already noticed from some users asking for ADL documentation. (_Debloater is alpha so no need for documentation_)
3. I don't expect all users to go through the documentation so carefully but I do think precautions are better so that we can point to them to read 1st TT.

**Comment by AbhiTheModder** [comment]
> Reasons:
> 1. I suggested here because this particular Tip&Tricks (TT) will be listed in top when AM release it's official AM_Community website.
> 2. When users will try versions other than **latest stable** they'll find many feature documentations missing and unnecessarily report on GH. e.g: I've already noticed from some users asking for ADL documentation. (_Debloater is alpha so no need for documentation_)
> 3. I don't expect all users to go through the documentation so carefully but I do think precautions are better so that we can point to them to read 1st TT.

Hmm... Got you point... Agreed👍

**Comment by MuntashirAkon** [comment]
Yes, I've had it in my mind since the beginning. But this is a low priority issue:
1. Versioning docs in App Manager isn't as critical as it is for Syncthing. Version-specific docs are also embedded within the app
2. Alpha/debug users should generally expect that the docs will not be up to date, and they need to follow changelogs extensively. Changelogs in App Manager is quite extensive compared to most other Android apps for a reason.

Regarding outdated docs, it has to be kept in mind that the entire production of App Manager is effectively run by a single person. So, this isn't quite unexpected. It would've been better if I received some help regarding the docs since it's not as complicated as developing the app itself.

**Comment by AbhiTheModder** [comment]
> Regarding outdated docs, it has to be kept in mind that the entire production of App Manager is effectively run by a single person. So, this isn't quite unexpected. It would've been better if I received some help regarding the docs since it's not as complicated as developing the app itself.

@MuntashirAkon If given the perms I can help with that in weekly basis. It may get delay for sometimes because of my exams schedules but it'll be sure the docs will be updated.

**Comment by MuntashirAkon** [comment]
> If given the perms I can help with that in weekly basis. It may get delay for sometimes because of my exams schedules but it'll be sure the docs will be updated.

You need to follow the CONTRIBUTING.rst file in the repo and create a PR with your changes. You may need to study some basic Latex syntax too since the docs are written using Latex.
